### PR TITLE
Update pipeline to deploy to test server

### PR DESCRIPTION
**Pull request by PlesnikJakub**
## Summary

Automate API deployment to one fixed DEV azure environment.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactoring (non-breaking change to code and structure)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: .github/workflows/dotnet-deploy.yml [hunk 1]
```diff
@@ -40,7 +40,8 @@ jobs:
       - name: Set up .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: ${{ env.DOTNET_VERSION }}
+          dotnet-version: 6.0.x
+          include-prerelease: true
```
Comment (fpindej) [inline .github/workflows/dotnet-deploy.yml:44]: Why use prerelease version? Latest stable should be the way to go imho. Keeping a project on a preview version when everyone is locally building on a stable version may be a potential cause of issues.

File: .github/workflows/dotnet-deploy.yml [hunk 2]
```diff
@@ -20,8 +20,8 @@
 name: Build and deploy ASP.Net Core app to an Azure Web App
 
 env:
-  AZURE_WEBAPP_NAME: interview-app                # set this to the name of your Azure Web App
-  AZURE_WEBAPP_PACKAGE_PATH: './src/WebApi'       # set this to the path to your web app project, defaults to the repository root
+  AZURE_WEBAPP_NAME: api-interview-dev            # set this to the name of your Azure Web App
```
Comment (fpindej) [inline .github/workflows/dotnet-deploy.yml:23]: If we're using this file for deploying the API to development environment, then I suppose we'll use a separate file for production deployment. Might consider renaming the .yml file 😊

**Review by fpindej — CHANGES_REQUESTED** [review]

File: .github/workflows/dotnet-deploy.yml (hunk 1 above)
Comment (petrspelos) [reply .github/workflows/dotnet-deploy.yml:44]: As far as I know, you still need to set `include-prerelease` to get .NET 6 in a pipeline. I know, sounds strange.

**Review by petrspelos — APPROVED** [review]
As long as @fpindej's comments are addressed, it's a go for me. I've seen it deploy properly with my own two eyes so... 😅

File: .github/workflows/dotnet-deploy.yml (hunk 1 above)
Comment (fpindej) [reply .github/workflows/dotnet-deploy.yml:44]: Where exactly does it cause problems? I'll use the most typical excuse called "works on my machine" 😅 But seriously, I don't have problems with not using it in my private repos 😕 

After doing a bit research on the internet, people seem to use it without it as well, see [here](https://dev.to/kritner/cicd-for-net-6-using-github-actions-1kln)

Comment (petrspelos) [reply .github/workflows/dotnet-deploy.yml:44]: Understandable, let's ditch it, then.

Comment (PlesnikJakub) [reply .github/workflows/dotnet-deploy.yml:44]: exactly as @petrspelos said, it's an old habit from times when it was required to mention prerelease :/ 
Thank you for noticing that :)

File: .github/workflows/dotnet-deploy.yml (hunk 2 above)
Comment (PlesnikJakub) [reply .github/workflows/dotnet-deploy.yml:23]: So far this is not an issue and dotnet-deploy is fine. We are far from production and there is an option to use the same workflow with different params. We will delay the decision on how to approach that.

**Review by petrspelos — APPROVED** [review]

**Review by fpindej — APPROVED** [review]

**Pull request merged**